app/schemas/com.kccreate.intelligentlockdown.streetpass.persistence.StreetPassRecordDatabase/1.json

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
should we be committing this file?

Reply:
I think so yeah! It's the database schema and I think this a change because of the package renaming that somehow wasn't picked up?